### PR TITLE
API-55 Document different environments

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Page about the test and production environments

### Changed

- Guides about user access tokens and client access tokens refer to the separate page about authorization server URLs

---

Ticket: https://jira.uitdatabank.be/browse/API-55

### CODE REVIEW

File: projects/authentication/docs/environments.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Authorization server URLs
+
+The authorization server to request [tokens](./methods.md) on is available on two domains, one for the test environment and one for the production environment of our APIs.
```
Comment (corneelwille) [inline projects/authentication/docs/environments.md:3]:
```suggestion
The authorization server to request [tokens](./methods.md) is available on two domains, one for the test environment and one for the production environment of our APIs.
```

**Review by corneelwille — APPROVED** [review]
Except for a small typo: clear and ready to merge!

File: projects/authentication/docs/environments.md (hunk 1 above)
Comment (bertramakers) [reply projects/authentication/docs/environments.md:3]: Hmm I meant it as in "requesting tokens on the server" -> "the server to request tokens on". But if it reads weird or is incorrect maybe it's better to reword the whole sentence 🤔

Comment (bertramakers) [reply projects/authentication/docs/environments.md]: @corneelwille can you check the change in [c8e359f](https://github.com/cultuurnet/apidocs/pull/64/commits/c8e359f519bc089da1082dda54c7eb53a550b8be)?

**Review by JonasVHG — APPROVED** [review]

**Review by corneelwille — APPROVED** [review]

**Pull request merged**